_includes/head.html

```diff
@@ -16,17 +16,17 @@
     <link href="/css/main.css" rel="stylesheet">
     
     <!-- Facebook metadata -->
-    <meta property="og:url" content="https://opensourcedesign.net">
-    <meta property="og:type" content="website">
-    <meta property="og:title" content="Open Source Design">
-    <meta property="og:image" content="https://opensourcedesign.net/images/opensourcedesign-logo.png">
-    <meta property="og:description" content="Connecting open source & design! 🎨">
-    <meta property="og:site_name" content="Open Source Design">
-    <meta property="og:locale" content="en_US">
+    <meta property="og:url" content="https://opensourcedesign.net" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="Open Source Design" />
+    <meta property="og:image" content="https://opensourcedesign.net/images/opensourcedesign-logo.png" />
+    <meta property="og:description" content="Connecting open source & design! 🎨" />
+    <meta property="og:site_name" content="Open Source Design" />
+    <meta property="og:locale" content="en_US" />
     
     <!-- Twitter metadata -->
     <meta name="twitter:card" content="summary">
-    <meta name="twitter:url" content="https://opensourcedesign.net">
+    <meta name="twitter:site" content="https://twitter.com/opensrcdesign" />
```

Comment:
Format is "<meta name="twitter:site" content="@nytimesbits" />" according to https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started

Reply:
ah I see